resources/views/Moment/index.blade.php

```diff
@@ -18,15 +18,15 @@
                   @foreach ($moments as $moment)
                   <tr>
                     <td><a href="{{route('moment.show', $moment)}}">{{$moment->title}}</a></td>
-                    <td><a href="{{route('moment.show', $moment)}}">{{$moment->briefDescription}}</a></td>
-                    <td><a href="{{route('moment.show', $moment)}}">{{$moment->procedure}}</a></td>
+                    <td><a href="{{route('moment.show', $moment)}}"></a>{!!$moment->briefDescription!!}</td>
+                    <td><a href="{{route('moment.show', $moment)}}"></a>{!!$moment->procedure!!}</td>
                     <td>
-                      <form  action="{{route('moment.destroy', $moment)}}" method="post">
+                      <form id="delete-moment{{$moment->title}}" action="{{route('moment.destroy', $moment)}}" method="post">
```

Comment:
En lugar de usar `id="delete-moment{{$moment->title}}"`, es mejor usar el `id`, porque sabes que nunca se va a repetir. deberia quedar algo como 
```
id="delete-moment-{{$moment->id}}"
```

Fijate que esto tambien lo usas en otros lugares. Hay que cambiarlo en todos.

Reply:
Cambiado en Moment y TO (ambos en index)